README.md

```diff
@@ -38,6 +38,18 @@ FLAGS:
 
 OPTIONS:
         --commit-sha <COMMIT HASH>                   Sets the hash of the commit used to generate the code coverage data
+        --excl-br-line <regex>
+            Lines in covered files containing this marker will be excluded.
```

Comment:
Help/docs about branch coverage exclusions should be probably be annotated differently, such as
... `Lines in covered files containing this marker will be excluded from branch coverage.` ?